packages/expo-font/src/__tests__/FontHooks-test.ts

```diff
@@ -17,6 +18,12 @@ describe('useFonts', () => {
     'ComicSans-Regular': 'path/to/jailed/font.ttf',
   };
 
+  beforeAll(() => {
+    if (Platform.OS !== 'web') {
+      NativeModulesProxy.ExpoFontLoader.getLoadedFonts.mockImplementation(() => []);
```

Comment:
Does this work properly? `NativeModulesProxy` is the old and deprecated way to expose native modules. Some tests are probably still relying on it though (calling it directly).

Reply:
it does. I took this from [previous code](https://github.com/expo/expo/blob/087046766cd9b7a7463c8eaee8daac8c5366d6af/packages/expo-font/src/__tests__/Font-test.native.ts#L32). What would be the new way to do this?

Reply:
Ok, let's leave it as is for now. See my comment here: https://github.com/expo/expo/blob/087046766cd9b7a7463c8eaee8daac8c5366d6af/packages/expo-font/src/__tests__/Font-test.native.ts#L42-L46
Generally our mocking system needs some love to work well with `globalThis.expo` instead 😅 Alek is currently working (https://github.com/expo/expo/pull/29870) on injecting modules to the global on Web for consistency and make it easier to improve our mocks (and get rid of `NativeModulesProxy`)

Reply:
leaving as-is